.opspec/release/to-ghcr/dood.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=$TARGETPLATFORM ghcr.io/linuxcontainers/alpine:3.11.3
+FROM --platform=$TARGETPLATFORM ghcr.io/linuxcontainers/alpine:3.18
```

Comment:
Unrelated; this is fixing the fact that this image isn't available in Github Container Registry (only identified via last build failing)